espanol/comida/victoria/victoria.md

```diff
@@ -1,3 +1,6 @@
 El capitán tras presentarse anter el rey, y mostrar su respeto con una gran reverencia.
 
-Le comunicó que su ejército había ganado la batalla decisiva contra el país vecino.
\ No newline at end of file
+Le comunicó que su ejército había ganado la batalla decisiva contra el país vecino.
+
+Pero la cosa no era tan sencilla. La guerra habia acabado con mas de la mitad de las tropas, dejando a Canudos en un estado de suma vulnerabilidad.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.